packages/mobile/src/screens/chat-screen/ReactionPopup.tsx

```diff
@@ -95,24 +99,47 @@ export const ReactionPopup = ({
     (r) => r.user_id === userIdEncoded
   )?.reaction
 
-  const beginAnimation = useCallback(() => {
-    Animated.spring(backgroundOpacityAnim.current, {
+  const beginMountAnimation = useCallback(() => {
+    Animated.timing(backgroundOpacityAnim.current, {
```

Comment:
Support using parallel, I think we could also stand to pull these out (plus the refs) into their own standalone hook to keep the components simpler